libs/registry/src/styles/style-luma.css

```
@@ -1240,6 +1240,35 @@
 		@apply min-w-0;
 	}
 
+	/* MARK: Message Scroller */
+	.spartan-message-scroller {
+		@apply relative flex size-full min-h-0 flex-col overflow-hidden;
+	}
+
+	.spartan-message-scroller-viewport {
+		@apply scroll-fade-b size-full min-h-0 min-w-0 overflow-y-auto overscroll-contain contain-content;
+		scrollbar-width: thin;
```

Stylelint (17.14.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)